### ISSUE TITLE
Update to 24.1.0

### ISSUE THREAD

**Issue by Mygod** [Gericop/Android-Support-Preference-V7-Fix#28]
Btw is #13 going to be merged some day?

**Comment by Mygod** [comment]
Oh great now this pops up in 24.1.0: (the code worked fine in 24.0.0)
```
Caused by: java.lang.IllegalStateException: Must specify preferenceTheme in theme
 at android.support.v14.preference.PreferenceFragment.onCreate(PreferenceFragment.java:218)
```

**Comment by Gericop** [comment]
I just updated my gradle file to 24.1.0 and it works for me without any further changes.

I don't know if #13 is going to be merged (*or rather directly put in code because merging would cause problems*) since it seems a bit of an overkill to me (custom layout, custom PreferenceTheme) compared to what you get (a scrollbar). Was this problem reported to Google?

**Comment by Mygod** [comment]
Try running it on API 16. That's where I tested it.

Well at least that's a solution. No I didn't.

**Comment by Gericop** [comment]
Report it to Google, please, they might fix it internally.

**Comment by Gericop** [comment]
I tried it in an API 16 emulator and it works fine.

**Comment by Mygod** [comment]
That's weird. Maybe it's because I'm inflating it in my layout file?

**Comment by Mygod** [comment]
I think it's because I'm using the v14 version of `PreferenceFragment`.

**Comment by Mygod** [comment]
The crash resolved itself in `24.1.1`.
